ms2query_app.py

```diff
@@ -80,3 +107,9 @@
 
 query_spectrums = [post_process_s2v(spec) for spec in query_spectrums]
 library_spectrums = [post_process_s2v(spec) for spec in library_spectrums]
+
+# turn spectra into documents
+documents_query = [SpectrumDocument(spec, n_decimals=2) for spec in
+                   query_spectrums]
+documents_library = [SpectrumDocument(spec, n_decimals=2) for spec in
+                     library_spectrums]
```

Comment:
That is the same as in #14 right? Just see my comment there on moving this part into an own function.